reV/supply_curve/sc_aggregation.py

```diff
@@ -365,11 +365,39 @@ def _parse_meta_cols(offshore_meta_cols, gen_meta):
 
         return offshore_meta_cols
 
+    @staticmethod
+    def _nearest_sc_points(excl_fpath, resolution, lat_lons):
+        """
+        Get nearest sc_points to offshore farms
+
+        Parameters
+        ----------
+        excl_fpath : str
+            Filepath to exclusions h5 with techmap dataset.
+        resolution : int
+            SC resolution
+        lat_lons : ndarray
+            Offshore wind farm coordinates (lat, lon) pairs
+
+        Returns
+        -------
+        points : pandas.DataFrame
+            Nearest SC points (gid, row_ind, and col_ind)
+        """
+        with SupplyCurveExtent(excl_fpath, resolution=resolution) as f:
+            points = f.points
+            sc_lat_lons = np.dstack((f.latitude, f.longitude))[0]
```

Comment:
Maybe have this 2D lat/lon array as a SCExtent property? Seems useful.

Reply:
done